partials/schemas/Address.yaml

```diff
@@ -0,0 +1,57 @@
+components:
+  schemas:
+    AbstractLocation:
```

Comment:
Maybe pull this out into a separate file to make it referable from other places as well. I assume it's used elsewhere, too.

Reply:
This also needs to be pulled out into a separate file